### PR TITLE
MODSOURMAN-779: Add status for Import jobs that are stopped by users.

### PR DESCRIPTION
**Pull request by VRohach**
## Purpose
Added new status "CANCELLED" for job that are stopped by users manually.

## Approach
Extended mechanism for status and ui-status execution

## Learning
https://issues.folio.org/browse/MODSOURMAN-779

### CODE REVIEW

**Comment by VRohach** [comment]
This PR can be merged only after merging: https://github.com/folio-org/data-import-raml-storage/pull/258

**Review by KaterynaSenchenko — APPROVED** [review]

File: mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json
Comment (RuslanLavrov) [inline mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json:195]: Hello @VRohach, please add ';' symbol at the end of the queries to fix their execution. Propose merging them into one line since they have the same purpose, in terms of changes to be introduced. Also, we don't need to change the "create_new_job_executions_table.sql" file because the value for status enums going to be added by scripts in the schema.json. Thank you!

**Review by SerhiiNosko — APPROVED** [review]

**Review by RuslanLavrov — APPROVED** [review]

**Comment by VRohach** [comment]
This branch is broken. So close this PR and created brand new one: https://github.com/folio-org/mod-source-record-manager/pull/628

**Pull request closed without merging**
